### ISSUE TITLE
Missing git tags for v2.02 and v2.03

### ISSUE THREAD

**Issue by andylolz** [IATI/IATI-Standard-SSOT#153]
[The SSOT page in the docs](http://iatistandard.org/203/developer/ssot/#finding-a-specific-version-of-the-ssot) says I can reference a particular version of the standard as it stood at release (e.g. what could be thought of as `2.03.0`) using tags. However, [that’s only the case up to v2.01](https://github.com/IATI/IATI-Standard-SSOT/tags).

The same is true for the component repositories, [e.g. the Schema repo](https://github.com/IATI/IATI-Schemas/tags).

**Comment by samuele-mattiuzzo** [comment]
I have been looking into this a bit, it seems like tags were a good idea at first but have been sort of misused: they do refer to the version of the standard the branch points to (`version-2.03` for instance) but they have not been kept up to date with the amount of changes, resulting in really old versions being tagged (2014-2015 for the 3 branches tagged). Unless we decide to only commit changes to the versions when a major/minor/patch update to the standard itself happens, they will just be mismatched and cause confusion (as we stand, for instance, we should tag `version-2.03` as `v2.03.64` -> does not match the Standard version and does not add a relevant piece of information). 

Rather than tagging them up every time, I'd suggest we drop the tags idea all together (it's not being used, afaik, anyway) and update the documentation.

The other idea would be to remove and reassign the tags every so often, which isn't a route I'm too keen on at the moment.

@Ocre42 and @andylolz thoughts on this?

**Comment by andylolz** [comment]
Worth noting that there was an accepted proposal to use semver starting from v3.0.0. At that point, I guess tags will probably become a better way of working, since no-one would want to have a branch for every point version, and especially given it’s exactly what tags are for.

---

When I created this ticket, I understood tags were used [as described at the end of the SSOT doc](http://reference.iatistandard.org/203/developer/ssot/#finding-a-specific-version-of-the-ssot) (linked above), i.e.:

>  * branches, which may change for typos and bugfixes e.g. `version-1.04`, see [_Branches_](http://reference.iatistandard.org/203/developer/ssot/proposal/#ssot-branches)
>  * tags, which point to the exact commit on the day of release e.g. `v1.04`, see [_Tags_](http://reference.iatistandard.org/203/developer/ssot/proposal/#ssot-tags)

I guess there could potentially be some use for this… But it’s not really idiomatic, and it’s potentially quite confusing (e.g. someone could be forgiven for thinking they should reference the tag, when 99 times out of 100 they probably should be tracking the version branch).

In summary: I guess (or, hope) most devs know to ignore tags and just use branches. Scrapping tags would probably be fine, except that when v3.0.0 happens, we might want to scrap branches and just use tags.

**Comment by samuele-mattiuzzo** [comment]
Ok, thanks for that piece of information I wasn't aware of.

Will scrape the tags and adjust the documentation accordingly, confirming deprecation of tags until semver kicks in.

**Comment by andylolz** [comment]
> Ok, thanks for that piece of information I wasn't aware of.

Which bit? The semver bit? Now I am doubting myself! Let me try and find a reference…

**Comment by samuele-mattiuzzo** [comment]
> Which bit? The semver bit? Now I am doubting myself! Let me try and find a reference…

Yes that one, although I think Dale mentioned something en passant about it when I met him before starting. If you can resurface the discussion that would be great, although not strictly needed (at least till we reach version `3.x.x`... we can talk about it at the TAG 😄 )

**Comment by andylolz** [comment]
> If you can resurface the discussion that would be great

Found it! It’s paper 10 from the [2017 Members’ Assembly in Rome](https://iatistandard.org/en/events/members-assembly-meeting-3-5-october-2017-rome/):

 * https://iatistandard.org/documents/242/Paper-10-Proposed-revisions-to-the-IATI-Standard-upgrade-process.pdf

**Comment by samuele-mattiuzzo** [comment]
Closed as, for now, all tags have been scraped
